Assets/MRTK/Providers/WindowsMixedReality/Shared/WindowsMixedRealityUtilities.cs

```diff
@@ -15,6 +16,7 @@
 #elif DOTNETWINRT_PRESENT
```

Comment:
```suggestion
#else
```

Reply:
I think I originally had this to make it more obvious what the `else` case was, but now that these have been cleaned up a bit I think this makes sense. Merged!